watertap/unit_models/osmotically_assisted_reverse_osmosis_base.py

```diff
@@ -507,15 +534,50 @@ def initialize_build(
             None
         """
 
-        source_flags = self.feed_side.initialize(
-            state_args=state_args,
+        init_log = idaeslog.getInitLogger(self.name, outlvl, tag="unit")
+        solve_log = idaeslog.getSolveLogger(self.name, outlvl, tag="unit")
+
+        # The model is isothermal, but the initialize methods
+        # (or the user) will fix the permeate-side temperature.
+        # Therefore, for initialization we need to deactivate
+        # the isothermal constraint but check that it will
+        # be enforced.
```

Comment:
I think these are previous comments, our current method does not need to deactivate the constraint, right?

Reply:
Removed in 86da522